src/frontend/src/views/GroupMembersPage/components/MembersTab/components/InviteModal/index.tsx

```diff
@@ -57,12 +53,30 @@ const InviteModal = ({
       // show a warning if we aren't sending invites for existing users
       const failedInvites = filter(invitations, (i) => !i.success);
       setFailedToSendAddresses(failedInvites.map((i) => i.email));
-      if (failedInvites.length > 0) setIsFailureNotificationOpen(true);
+      if (failedInvites.length > 0) {
+        dispatch(addNotification({
+          reduxId: Date.now(),
+          shouldShowCloseButton: true,
+          intent: "warning",
+          componentProps: {
+            failedToSendAddresses,
+          }
+        }));
+      }
 
       // show success for any invites we did send
       const successCount = invitations.length - failedInvites.length;
       setSentCount(successCount);
-      if (successCount > 0) setIsSuccessNotificationOpen(true);
+      if (successCount > 0) {
+        dispatch(addNotification({
```

Comment:
There's no `reduxId` on this one -- is that a problem, or does the `autoDismiss` make things okay? Would it prevent an early dismiss because it doesn't have a reduxId?

Reply:
it would prevent the notif from being dismissed. just added